app/views/reviews/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= form_with(model: review, local: true, url: restaurant_reviews_path) do |form| %>
+  <% if review.errors.any? %>
+    <div id="error_explanation">
```

Comment:
i would have removed all of this div to a different partial, in order to keep this view clean